challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -7761,6 +7761,724 @@
+        },
+        {
+          "text":
+            "'<code>ludic(20)</code> should return <code>'+JSON.stringify(results[3])+'</code>.'",
+          "testString":
+            "assert.deepEqual(ludic(20),results[3],'<code>ludic(20)</code> should return <code>'+JSON.stringify(results[3])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>ludic(26)</code> should return <code>'+JSON.stringify(results[4])+'</code>.'",
+          "testString":
+            "assert.deepEqual(ludic(26),results[4],'<code>ludic(26)</code> should return <code>'+JSON.stringify(results[4])+'</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7efc",
+      "challengeType": 5,
+      "releasedOn": "July 10, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function ludic (n) {", "  // Good luck!", "}"],
+          "head": [],
+          "tail": [
+            "var results=[",
+            "[1,2],",
+            "[1,2,3],",
+            "[1,2,3,5],",
+            "[1,2,3,5,7,11,13,17],",
+            "[1,2,3,5,7,11,13,17,23,25]]"
+          ]
+        }
+      }
+    },
+    {
+      "title": "Luhn test of credit card numbers",
+      "description": [
+        "The <a href=\"https://en.wikipedia.org/wiki/Luhn algorithm\">Luhn test</a> is used by some credit card companies to distinguish valid credit card numbers from what could be a random selection of digits.",
+        "Those companies using credit card numbers that can be validated by the Luhn test have numbers that pass the following test:",
+        "<ol><li> Reverse the order of the digits in the number.</li>",
+        "<li> Take the first, third, ... and every other odd digit in the reversed digits and sum them to form the partial sum s1</li>",
+        "<li> Taking the second, fourth ... and every other even digit in the reversed digits:</li></ol>",
+        "#  Multiply each digit by two and sum the digits if the answer is greater than nine to form partial sums for the even digits",
+        "#  Sum the partial sums of the even digits to form s2",
+        "If s1 + s2 ends in zero then the original number is in the form of a valid credit card number as verified by the Luhn test.",
+        "///",
+        "For example, if the trial number is 49927398716:",
+        "Reverse the digits:",
+        "61789372994",
+        "Sum the odd digits:",
+        "6 + 7 + 9 + 7 + 9 + 4 = 42 = s1",
+        "The even digits:",
+        "1,  8,  3,  2,  9",
+        "Two times each even digit:",
+        "2, 16,  6,  4, 18",
+        "Sum the digits of each multiplication:",
+        "2,  7,  6,  4,  9",
+        "Sum the last:",
+        "2 + 7 + 6 + 4 + 9 = 28 = s2s1 + s2 = 70 which ends in zero which means that 49927398716 passes the Luhn test",
+        "Write a function that will validate a number with the Luhn test. Return true if valid else false."
```

Comment:
Let's just change this a little:

"Return true if it's a valid number. Otherwise, return false."